frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -1001,6 +1002,26 @@ export const ClusterVersionDetailsTable: React.FC<ClusterVersionDetailsTableProp
             <dd>
               <ResourceLink kind={referenceForModel(ClusterVersionModel)} name={cv.metadata.name} />
             </dd>
+            <dt>{t('public~Upstream configuration')}</dt>
+            <dd>
+              <div>
+                {!cv.spec.upstream ? (
+                  <span className="text-muted">{t('public~Default configuration: ')}</span>
+                ) : null}
+                <Button
```

Comment:
@wking @alimobrem Do you know if we need to disable this for OpenShift Dedicated? I don't know if Dedicated admins can edit the ClusterVersion.

@florkbr We need to add an access review here and only show the button if the user can patch the ClusterVersion resource. Here's an example:

https://github.com/jamestalton/ocp-console/blob/e12a4adb5c4bde317d510a0590c88f6b99c2f6d6/frontend/public/components/cluster-settings/cluster-settings.tsx#L848-L854

Reply:
I've added the access control check.